### PR TITLE
implement timestamp macros

### PR DESCRIPTION
**Pull request by MartinNowak**

### CODE REVIEW

File: source/ddox/ddoc.d [hunk 1]
```diff
@@ -19,7 +19,7 @@ import std.uni : isAlpha;
 // TODO: support escapes section
 
 
-static this()
+shared static this()
```
Comment (s-ludwig) [inline source/ddox/ddoc.d]: `s_standardMacros` would have to be made `__gshared` for this to work, but why not.

Comment (s-ludwig) [inline source/ddox/ddoc.d]: Or rather, I've just discovered that `immutable` works, too - nice!

**Comment by MartinNowak** [comment]
Fixed the remaining issues. Someone should write the missing date formatting code for std.datetime ;).

**Comment by s-ludwig** [comment]
Yeah, I've missed that a number of times, too. I guess something along the lines of what Linux `date` uses would be a good choice.

**Pull request merged**